pkg/monitor/datapath_drop.go

```diff
@@ -80,7 +80,7 @@ func DecodeDropNotify(data []byte, dn *DropNotify) error {
 
 func (n *DropNotify) decodeDropNotify(data []byte) error {
 	if l := len(data); l < dropNotifyV1Len {
-		return fmt.Errorf("unexpected DropNotify data length, expected %d but got %d", dropNotifyV1Len, l)
+		return fmt.Errorf("unexpected DropNotifyV1 data length, expected %d but got %d", dropNotifyV1Len, l)
```

Comment:
Even though we're testing against `dropNotifyV1Len` we do not yet know whether the payload is v1 or v2, so how about not specifying the version in this error message:
```suggestion
		return fmt.Errorf("unexpected DropNotify data length, expected at least %d but got %d", dropNotifyV1Len, l)
```
We can still differentiate with the one testing against `dropNotifyV2Len`.